src/sentry/dynamic_sampling/rules/data/key_transactions.py

```diff
@@ -1,7 +1,7 @@
 from typing import TYPE_CHECKING, List
```

Comment:
Entire `data` folder confuse me a lot:
- data sounds like data folder with data
- IMO it's just 1 file per func, so i prefer to move it to utils may be?
I understand the point you are trying to isolate later of data ... but TBH it do nothing, each bias has different data input, also we don't introduce any new abstract classes here. 🤔